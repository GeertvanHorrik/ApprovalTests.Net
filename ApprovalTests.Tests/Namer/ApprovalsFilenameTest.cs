using ApprovalTests.Namers;
using NUnit.Framework;

namespace ApprovalTests.Tests.Namer
{
    public class ApprovalsFilenameTest
    {
        [Test]
        public void TestMachineSpecificName()
        {
            var approvalsFilename = ApprovalsFilename.Parse(@"C:\Users\olgica\Documents\GitHub\ApprovalTests.Net\ApprovalTests.Tests\Email\EmailTest.Testname.Microsoft_Windows_10_Education.approved.eml");
            Approvals.Verify(approvalsFilename);
            Assert.True(approvalsFilename.IsMachineSpecific);
        }

        [Test]
        public void TestNonMachineSpecificName()
        {
            Approvals.Verify(ApprovalsFilename.Parse(@"C:\Users\olgica\Documents\GitHub\ApprovalTests.Net\ApprovalTests.Tests\Email\EmailTest.Testname.approved.eml"));
        }
    }
}