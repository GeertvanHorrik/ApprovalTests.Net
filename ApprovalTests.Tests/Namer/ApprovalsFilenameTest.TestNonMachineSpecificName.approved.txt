GetFullPath: C:\Users\olgica\Documents\GitHub\ApprovalTests.Net\ApprovalTests.Tests\Email\EmailTest.Testname.approved.eml
Directory: C:\Users\olgica\Documents\GitHub\ApprovalTests.Net\ApprovalTests.Tests\Email
ClassName: EmailTest
MethodName: Testname
AdditionalInformation: []
ApprovedStatus: approved
Extension: eml